connector/authproxy/authproxy.go

```diff
@@ -14,16 +14,24 @@ import (
 
 // Config holds the configuration parameters for a connector which returns an
 // identity with the HTTP header X-Remote-User as verified email.
-type Config struct{}
+type Config struct {
+	HeaderName string   `json:"headerName"`
```

Comment:
I wonder if this should be called something like `UserHeaderName` or just `UserHeader`. Another common header name is `X-Forwarded-Groups` for group information.